Client/Frontend/Browser/TabManager.swift

```diff
@@ -510,6 +516,23 @@ extension TabManager : WKNavigationDelegate {
         }
         UIApplication.sharedApplication().networkActivityIndicatorVisible = false
     }
+
+    /// Called when the WKWebView's content process has gone away. If this happens for the currently selected tab
+    /// then we immediately reload it. If this happens for a tab that is in the background, we mark the tab as
+    /// terminated so that we can reload it when it is activated.
+
+    func webViewWebContentProcessDidTerminate(webView: WKWebView) {
+        for tab in tabs {
```

Comment:
`TabManager` has a `subscript(webView: WKWebView) -> Browser?` -- could we reuse that?